themes/bootstrap5/js/record.js

```diff
@@ -2,7 +2,10 @@
 /*exported ajaxTagUpdate, recordDocReady, refreshTagListCallback, addRecordRating */
 
 /**
- * Functions and event handlers specific to record pages.
+ * Check if a user request is valid by making an AJAX call.
+ * @param {HTMLElement} element The link element to check.
+ * @param {string} requestType The type of request (e.g., 'Hold', 'StorageRetrievalRequest').
+ * @param {string} [icon='place-hold'] The icon to display.
```

Comment:
I haven't seen this syntax elsewhere. Is this valid, or should it be the below for consistency with other places?

```suggestion
 * @param {string} [icon] The icon to display (default = place-hold).
```

Reply:
I was following our regular syntax for most of the other docs. However, I noticed the [official documentation](https://jsdoc3.vercel.app/tags/param) suggested this syntax, so I wanted to get your thoughts on which one we should use.

Reply:
Since this is officially documented, I think it does make sense to use this format! The examples seem to suggest that perhaps the single quotes are not needed... though if you omit the quotes, I'm not sure how you're supposed to differentiate between a hard-coded value and a variable or constant name. Maybe that's just inferred based on the type. If you'd like to make the default syntax consistent in this PR, please feel free (in other words, adjust the document default below to use this same format for consistency). If you want to go back and open a PR to reformat the existing `(default = xxx)` comments to this format as well, I'd be happy to merge that too.